### PR TITLE
Add Workflow to README

### PR DESCRIPTION
**Pull request by brminnick**
### Description of Change ###

This PR updates our README to add the Contributor Maintainer Workflow.

Please double check the workflow for accuracy and typos! 

If possible, I'd like to get approval from all of the core maintainers before merging this PR ❤️ 

NET MAUI Workflow

### CODE REVIEW

**Review by pictos — APPROVED** [review]
LGTM❣️

**Review by jfversluis — DISMISSED** [review]
Looks good to me! Let's see if this is also clear for outside people that the bottom actions actually match the top ones. Maybe we could make things even clearer with some arrow action.

For maintainability we could consider Mermaid charts? Then the source of this also lives in the repo... But I think this is a great start!

File: README.md [hunk 1]
```diff
@@ -75,7 +77,7 @@ Every new feature also requires an associated sample to be added to the .NET MAU
 ### 6. Documentation Complete 
 Before a Pull Request can be merged into the .NET MAUI Toolkit, the Pull Request Author must also submit the documentation to [documentation repository](https://github.com/MicrosoftDocs/CommunityToolkit).
 
-### 7. Merged
+### 7. Completed
```
Comment (jfversluis) [inline README.md:80]: This is now different than from what's in the chart. Isn't that confusing? Maybe also make the rest of the headings reflect the chart more?

File: README.md [hunk 2]
```diff
@@ -41,7 +41,9 @@ xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
 
 ## Submitting A New Feature
 
-New features will follow the following steps:
+New features will follow the below workflow, described in more detail in the steps below
+
+![New Feature Workflow](https://user-images.githubusercontent.com/13558917/160728690-a0e206f5-d37e-472e-a1bd-8fe686eb37fb.png)
```
Comment (VladislavAntonyuk) [inline README.md:46]: maybe you can add the source of the image to the .github folder, if the flow changes in the future, we can easily update it.

**Review by VladislavAntonyuk — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (brminnick) [reply README.md:80]: Ah - good catch! The [Project Column](https://github.com/CommunityToolkit/Maui/projects/1) is `Completed`, so I'll update the drawing to be `Completed` as well.

Long story short, all of the `msft-bot` automations are based around the Project Column names, so it's easier for us to change the docs to reflect the project columns than it is to change the Project Column names.

File: README.md (hunk 2 above)
Comment (brminnick) [reply README.md:46]: Will do!

**Comment by brminnick** [comment]
NET MAUI Workflow

Thanks all! I've updated the chart, stealing Vlad's great idea for the arrows and fixing the typo that Gerald pointed out.

**Comment by brminnick** [comment]
For now, I've added the `.sketch` source file to the `/build/` folder. 

I plan on migrating this chart to Mermaid, but I don't have the bandwidth this week to learn + implement Mermaid.  I'll open a Discussion to remind me to do it, though 👍

**Comment by soyfrien** [comment]
Thanks for making it. Here is an SVG if you want something in the meantime: https://gist.github.com/soyfrien/3932e1d648b54c3b30970f8367615a6a

**Pull request merged**